packages/geoadmin-tooltip/src/GeoadminTooltip.vue

```diff
@@ -232,123 +219,45 @@ defineExpose({ tooltipElement, openTooltip, closeTooltip })
                     {{ tooltipContent }}
                 </slot>
 
-                <!-- the arrow to be displayed on the bottom or on the right-->
                 <div
-                    v-if="placement == 'top' || placement == 'left'"
-                    ref="arrowDownLeftElement"
-                    :style="arrowStyle"
-                    class="arrow"
-                    :class="{ arrowDown: placement == 'top', arrowRight: placement == 'left' }"
+                    v-if="middlewareData.arrow"
+                    ref="arrowElement"
+                    :style="{
+                        left: `${middlewareData.arrow.x}px`,
+                        top: `${middlewareData.arrow.y}px`,
+                    }"
+                    class="tw:absolute tw:h-0 tw:w-0 tw:after:content-[''] tw:after:h-0 tw:after:w-0 tw:after:fixed tw:after:top-[inherit] tw:after:left-[inherit] tw:after:z-[-1]"
+                    :class="{
```

Comment:
Maybe it could make sense to move this into a `computed` to keep the HTML part less cluttered? I think it is rather easy to construct arrays or objects that will the be passed to the attribute

Reply:
I really don't like when we mix our JS code with HTML class generation, that's why I tried to kept everything related to Tailwind described in the template section.

It could be improved a bit if moved to JS, but as I said, I prefer to keep the JS doing its part (interaction, reactivity, etc) and let the template be responsible for the styling